packages/wallets/src/metamask/pages/onboarding.page.ts

```diff
@@ -0,0 +1,76 @@
+import { Locator, Page, test } from '@playwright/test';
+import { WalletConfig } from '../../wallets.constants';
+
+export class OnboardingPage {
+  page: Page;
+  termsCheckboxButton: Locator;
+  importWalletButton: Locator;
+  metricAgreeButton: Locator;
+  secretPhraseInputs: Locator;
+  secretPhraseImportButton: Locator;
+  createPasswordInput: Locator;
+  confirmPasswordInput: Locator;
+  createPasswordTerms: Locator;
+  createPasswordImport: Locator;
+  completeButton: Locator;
+  pinExtensionNextButton: Locator;
+  pinExtensionDoneButton: Locator;
+
+  constructor(page: Page, public config: WalletConfig) {
+    this.page = page;
+    this.termsCheckboxButton = this.page.getByTestId(
+      'onboarding-terms-checkbox',
+    );
+    this.importWalletButton = this.page.getByTestId('onboarding-import-wallet');
+    this.metricAgreeButton = this.page.getByTestId('metametrics-i-agree');
+    this.secretPhraseInputs = this.page.locator(
+      '.import-srp__srp-word >> input[type=password]',
+    );
+    this.secretPhraseImportButton = this.page.getByTestId('import-srp-confirm');
+    this.createPasswordInput = this.page.getByTestId('create-password-new');
+    this.confirmPasswordInput = this.page.getByTestId(
+      'create-password-confirm',
+    );
+    this.createPasswordTerms = this.page.getByTestId('create-password-terms');
+    this.createPasswordImport = this.page.getByTestId('create-password-import');
+    this.completeButton = this.page.getByTestId('onboarding-complete-done');
+    this.pinExtensionNextButton = this.page.getByTestId('pin-extension-next');
+    this.pinExtensionDoneButton = this.page.getByTestId('pin-extension-done');
+  }
+
+  async firstTimeSetup() {
+    await test.step('First time setup', async () => {
+      await this.confirmTermsOfOnboarding();
+      await this.importWalletButton.click();
+      await this.metricAgreeButton.click();
+      await this.fillSecretPhrase(this.config.SECRET_PHRASE);
+      await this.secretPhraseImportButton.click();
+      await this.createPassword(this.config.PASSWORD);
+      await this.completeButton.click();
+      await this.pinExtensionNextButton.click();
+      await this.pinExtensionDoneButton.click();
+      await this.page.waitForURL('**/home.html#');
+    });
+  }
+
+  async confirmTermsOfOnboarding() {
+    while (!(await this.page.locator('.check-box__checked').isVisible())) {
+      console.log('Checkbox is not checked');
```

Comment:
we really need this log?

Reply:
Half year ago, I added that log for wathing when that flaky checkbox will be working correct.
But we still get logs like this 
<img width="229" alt="image" src="https://github.com/user-attachments/assets/4b46a4f9-9b1c-40b5-aac6-1ee6bb670ec4">

I wanna to add more info as like `console.error('confirmTermsOfOnboarding function: Checkbox is not checked (checkbox enable state = ${await this.termsCheckboxButton.isEnabled()})')` to continue watching